### PR TITLE
feat: ES modal, instrumentation, and func tests

### PR DESCRIPTION
**Pull request by danzhaas**

### CODE REVIEW

File: demo/standalone.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
         <title>Messaging.js Dev Sandbox</title>
         <meta charset="UTF-8" />
         <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
-        <script src="//localhost.paypal.com:8080/messaging.js" data-pp-account="DEV00000000NI"></script>
+        <script src="//localhost.paypal.com:8080/messaging.js" data-pp-account="FZUV5T4AD6C4A"></script>
```
Comment (gillycheesesteak) [inline demo/standalone.html]: Can we revert this change please

File: src/components/modal/content/ES/parts/ContentWrapper.jsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
+/** @jsx h */
```
Comment (gillycheesesteak) [inline src/components/modal/content/ES/parts/ContentWrapper.jsx:1]: Shouldn't these files (`ES/parts/ContentWrapper.jsx`, `ES/parts/GPL.jsx` and the new css files) be unnecessary with universal modal?

**Comment by danzhaas** [comment]
This PR being closed in favor of #733 due to more straightforward development now that universal modal feature branch was merged into develop.
